bots/zephyr_mirror_backend.py

```diff
@@ -740,6 +769,7 @@ def maybe_forward_to_zephyr(message):
             logger.exception("Error forwarding message:")
 
 def zulip_to_zephyr(options):
+    # type: (zulip) -> None
```

Comment:
So `options` which is something returned by `OptionParser.parse` if of class `zulip`? Remember `OptionParser` is in the standard library, so it has no idea of `zulip`.